src/components/dialog/DeleteFileDialog.vue

```diff
@@ -3,21 +3,21 @@
     dialog-key="DeleteFile"
     data-cy="delete-file-dialog"
   >
-    <template v-slot:title>
+    <template #title>
       <q-icon
         color="primary"
         :name="isFolder ? 'fa-solid fa-folder-minus' : 'fa-solid fa-file-circle-minus'"
       />
       {{ $t(`page.modelizer.fileExplorer.delete.${ isFolder ? 'folder' : 'file' }.title`) }}
     </template>
-    <template v-slot:default>
-      <div
-        class="q-pb-lg"
-        v-html="$t(
-          `page.modelizer.fileExplorer.delete.${ isFolder ? 'folder' : 'file' }.description`,
-          { name: deletedFile.label },
-        )"
-      >
+    <template #default>
+      <div class="q-pb-lg">
+        {{
+          $t(
+            `page.modelizer.fileExplorer.delete.${ isFolder ? 'folder' : 'file' }.description`,
+            { name: deletedFile.label },
+          )
+        }}
```

Comment:
since no longer v-html, have you checked the translation ? 
`'Are you sure you want to delete <b>{name}</b> file ?'`

Reply:
I disable the rules and restore the v-html